packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/adding-basic-auth/index.md

```diff
@@ -15,7 +15,9 @@ To add the `npm` packages:
 
 After including the packages, add the following code snippet in your project.
```

Comment:
I don't have the full page for context, but having the "add the following code snippet in your project" and then adding two additional paragraphs of text would create confusion, so maybe this line should go after the two paragraphs (directly before the snipped stack selector)?

Reply:
Yes. Good catch. Thanks.